apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -28,7 +28,7 @@ var ColorPickerPropertyRow = React.createClass({
    * Make our button a colpick color picker, if it isn't already
    */
   ensureColorPicker: function () {
-    var element = React.findDOMNode(this.refs.colorPicker);
+    var element = ReactDOM.findDOMNode(this.refs.colorPicker);
```

Comment:
Ahhh, TIL. I guess that maybe makes sense though. Good to know.

On Wed, Mar 2, 2016 at 11:16 AM, Brad Buchanan <notifications@github.com>
wrote:

> In apps/src/applab/designElements/ColorPickerPropertyRow.jsx
> <https://github.com/code-dot-org/code-dot-org/pull/7083#discussion_r54774709>
> :
>
> > @@ -28,7 +28,7 @@ var ColorPickerPropertyRow = React.createClass({
> >     * Make our button a colpick color picker, if it isn't already
> >     */
> >    ensureColorPicker: function () {
> > -    var element = React.findDOMNode(this.refs.colorPicker);
> > +    var element = ReactDOM.findDOMNode(this.refs.colorPicker);
>
> Ah! My change of all 12 instance broke tests, and I figured out why: Refs
> to custom (user-defined) components work exactly as before; only the
> built-in DOM components are affected by this change.
> <https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs>
>
> So this particular case, where refs.colorPicker is a <button> we can
> replace this, but in some cases (like refs.age in ShareWarnings.jsx) we
> still have to use findDOMNode because the ref is on a user-defined
> component.
>
> Fixing...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/7083/files#r54774709>.
>